### ISSUE TITLE
Warsong Gulch flag appearing in front of the player when entering the bg

### ISSUE THREAD

**Issue by Sabre142** [elysium-project/server#2522]
![image](https://user-images.githubusercontent.com/26516348/30264638-b0e5d97a-971c-11e7-9196-734207d3b600.png)

It is clickable but nothing happens, and it soon fades away.

**Comment by Itembugs** [comment]
Wonder whats the cause

**Comment by Seanio1080** [comment]
yeah this needs to be fixed. its so obvious im surprised it hasnt been a priority
